src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -379,6 +379,14 @@ public partial class InvalidMessageContractException : System.Exception
         public InvalidMessageContractException(string message) { }
         public InvalidMessageContractException(string message, System.Exception innerException) { }
     }
+
+    [System.AttributeUsageAttribute((System.AttributeTargets)(384), AllowMultiple = false, Inherited = false)]
```

Comment:
Other APIs are already using this magic number. I think we should just make it consistent with other types.